apps/admin/backend/src/adjudication.test.ts

```diff
@@ -238,4 +238,14 @@ test('adjudicateWriteIn', async () => {
       status: 'invalid',
     }
   );
+
+  await adjudicateWriteIn({ writeInId, type: 'reset' }, store, logger);
+  expectVotes({ 'zoo-council-mammal': ['write-in-0'] });
+  expectWriteInRecord({
+    status: 'pending',
+  });
+  expectLog(`User adjudicated a write-in from invalid to unadjudicated.`, {
```

Comment:
This log could be phrased better but don't think it's important enough for now to complicate the logic.